app/views/components/fullWidthMainContent.scala.html

```diff
@@ -19,7 +19,7 @@
 @(contentBlock: Html)
 
 <div class="govuk-grid-row">
- <div class="govuk-grid-column-full">
+ <div class="govuk-grid-column-two-thirds">
   @contentBlock
```

Comment:
![image](https://github.com/user-attachments/assets/ca8ba7c6-5e5f-4cf0-a1d1-10a4a39193e1)
This makes the table layout two third. Issue is with the length of the word.

Reply:
Thanks, yeah after speaking with Lydia the table needs to be full-width, so I think the way it was before was correct